products/opensuse/main.pm

```diff
@@ -143,6 +141,20 @@ sub is_server() {
     return (get_var("OFW") || check_var("FLAVOR", "Server-DVD"));
 }
 
+sub is_tumbleweed {
+    # Tumbleweed and its stagings
+    return 0 unless check_var('DISTRI', 'opensuse');
```

Comment:
I guess this pointless in products/opensuse - but I felt better with having this extra backup :)